src/assets/icons/Done.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import DoneIcon from './done.png';
+
+interface OwnProps {}
+
+export function Done(props: OwnProps) {
+  return <img {...props} alt="done icon" src={DoneIcon} />;
```

Comment:
use i18n for alt prop,

```suggestion
  return <img {...props} alt={i18n._('Done')} src={DoneIcon} />;
```